railties/test/generators/app_generator_test.rb

```diff
@@ -695,6 +695,24 @@ def test_kamal_storage_volume_is_skipped_if_required
     end
   end
 
+  def test_gitignore_skips_storage_directory_if_active_storage_is_skipped_and_database_is_sqlite
```

Comment:
I think this name is confusing, not sure what to recommend though :thinking:

Reply:
I'll try thinking about a better name